### PR TITLE
Update phpunit development

### PR DESCRIPTION
**Pull request by rachadiannovansyah**
image
image

Overview :
- Up coverage
- Fixing failure test
- Setup phpunit.xml

Note:
- Beberapa fungsi mungkin blm kena hit phpunit karna butuh riset lagi
- Current Coverage 79.83%

### CODE REVIEW

**Comment by rindibudiaramdhan** [comment]
masih faiing laravel coveragenya @rachadiannovansyah ?

**Comment by rachadiannovansyah** [comment]
> masih faiing laravel coveragenya @rachadiannovansyah ?

lagi solving kang

File: app/Http/Controllers/V1/ReservationController.php [hunk 1]
```diff
@@ -77,7 +77,7 @@ public function store(ReservationRequest $request)
             'approval_status' => ReservationStatusEnum::already_approved(),
         ]);
 
-        Mail::to(config('mail.admin_address'))->send(new ReservationStoreMail($reservation));
+        // Mail::to(config('mail.admin_address'))->send(new ReservationStoreMail($reservation));
```
Comment (ayocodingit) [inline app/Http/Controllers/V1/ReservationController.php:80]: kenapa di comment kang @rachadiannovansyah

Comment (rachadiannovansyah) [reply app/Http/Controllers/V1/ReservationController.php:80]: waduh lagi testing phpunit kang, ini dibalikin ko di commitan yg baru 😅👍

**Review by ayocodingit — APPROVED** [review]

**Review by rindibudiaramdhan — APPROVED** [review]

**Pull request merged**